src/components/pages/MatchUp/MatchUpContainer.js

```diff
@@ -102,6 +102,7 @@ export default connect(
     child: state.child,
     faceoffs: state.faceoffs,
     votes: state.votes,
+    custom_date: state.date.custom_date,
```

Comment:
Is this file actually using the custom_date?

Reply:
Yes, this is the parent component for `RenderMatchUp` and `FaceoffContent`. By adding `custom_date` to the parent component, the data will be passed to its children as props, so we won't need to connect each child to the Redux. The main reason I went with this approach is that we get the same result with the minimum amount of code.